new-lamassu-admin/src/styling/icons/arrow/carousel-left-arrow.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="13px" height="33px" viewBox="0 0 13 33" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Simple Arrow White</title>
```

Comment:
This `title` node makes its content show up during mouseover icon. I believe you can delete it to keep the behavior similar to the other icons